data/EnzymaticCoreactants.tsv

```diff
@@ -0,0 +1 @@
+#ID 	Name 	SMILES
```

Comment:
Might want to call this NoCoreactants so that it's not confused with the file in pickaxe/data of the same name

Reply:
good idea